_entries/02 lab rating app.md

```diff
@@ -1,9 +1,9 @@
 ---
 sectionid: lab-ratingapp
 sectionclass: h1
-title: Lab 1 - Rating App
+title: Lab 1 - Go Microservices
```

Comment:
Make the lab title be more intuitive. This lab is focusing on showing how to deploy and run microservices on ARO.